.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+## **BEFORE YOU SUBIMIT** please read the following:
+If you have a support request or question please 
+submit them to [StackOverflow](http://stackoverflow.com/questions/tagged/webpack) using the tag `[webpack]` or the [webpack Gitter](https://gitter.im/webpack/webpack). Future support requests will be closed.
+
+
+
+I'm submitting a...
+=====
+- [ ] bug report
+- [ ] feature request
+- [ ] support request => Please do not submit support request here, see note at the top of this template. 
+
+
+** Webpack version: **
```

Comment:
`** Webpack version: **` doesn't render right because of the spaces inside the asterisks:

** Webpack version: **